docs/guide/marko-5-interop.md

```
@@ -1,3 +1,94 @@
+```marko
+// use class
+<h1>Class API</h1>
+```
+
+```marko
+<!-- use tags -->
+<h1>Tags API</h1>
+```
+
+> [!TIP]
+> These explicit opt-ins are only necessary if a `.marko` file _isn't_ an [auto-discovered tag](#components-vs-tags) and its contents are ambiguous (i. e. none of the following heuristics apply) so in practice they are rarely needed except for sometimes in [Marko Run](../marko-run/file-based-routing.md)'s `+page.marko` and `+layout.marko`.
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)